<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirDVD</class>
 <widget class="QDialog" name="IncluirDVD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO DVD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_nome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_preco">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preço:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_duracao">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_preco">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_duracao">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>180</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IncluirDVD</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
